driver/config/config.go

```diff
@@ -67,6 +67,8 @@ const (
 	ViperKeyCourierTemplatesPath                             = "courier.template_override_path"
 	ViperKeyCourierTemplatesRecoveryInvalidEmail             = "courier.templates.recovery.invalid.email"
 	ViperKeyCourierTemplatesRecoveryValidEmail               = "courier.templates.recovery.valid.email"
+	ViperKeyCourierTemplatesRecoveryCodeInvalidEmail         = "courier.templates.recovery_code.invalid.email"
+	ViperKeyCourierTemplatesRecoveryCodeValidEmail           = "courier.templates.recovery_code.valid.email"
 	ViperKeyCourierTemplatesVerificationInvalidEmail         = "courier.templates.verification.invalid.email"
 	ViperKeyCourierTemplatesVerificationValidEmail           = "courier.templates.verification.valid.email"
```

Comment:
Hm, probably good to not break BC here. At the same time it would be better to have:

```patch
courier:
  templates:
    recovery:
-      valid:
-      invalid:
+      link:
+        valid:
+        invalid:
+      code:
+        valid:
+        invalid:
```

Reply:
Maybe we can use

```patch
courier:
  templates:
    recovery:
      valid:
      invalid:
+     code:
+       valid:
+       invalid:
```

It's a bit awkward as long as the `link` strategy is still there. But once that's gone should be good, right?

Reply:
That's ok. It will prevent us from having the same problem again next time we introduce a new recovery template.

Reply:
So you're fine with my proposal?

Reply:
I think @zepatrik resolved this by accident, this is not done.